oneanddone/users/views.py

```diff
@@ -93,6 +96,7 @@ def get_context_data(self, **kwargs):
         context = super(ProfileDetailsView, self).get_context_data(**kwargs)
         context['attempts_finished'] = attempts_finished
         context['page'] = 'ProfileDetails'
+        context['activity_url'] = activity_url
```

Comment:
If a user has not entered a `bugzilla_email` then `activity_url` will not have been set yet, and this will throw an exception.